tests/dummy/app/routes/tests/components/destroyed-container/template.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if showContainer}}
+  {{#flexi-container}}
+    template block text
+  {{/flexi-container}}
+{{/if}}
+
+{{#component "tests/components/destroyed-container/components/component-that-destroys-container" showContainer=(mut showContainer)}}
```

Comment:
removed the component helper from here and the test itself, though the names are still a little longer than hoped for.